### DISCUSSION TITLE
Generate local NuGet package on MacOS

### DISCUSSION THREAD

**Discussion by sid-dahiya** [AvaloniaUI/Avalonia#6277] [Q&A] [answered]
How can I generate a local NuGet package on MacOS?

I'm trying to validate #5177 locally by integrating it in our app and have validated that the test app (ControlCatalog.NetCore) is working correctly. As the next step, I'd like to test out the integration in our app, for which I need to generate nuget package locally I can update our projects to.

CC: @ShrutiJaiswal1494

**Comment by kekekeks** [comment, marked as answer]
`./build.sh --target CreateNugetPackages --configuration=Release`

**Reply by sid-dahiya** [reply]
Thank you!
